docs/content/manual/release-specific/v1.2.0/test-backing-image-upload.md

```diff
@@ -14,6 +14,24 @@ title: Test backing image
 9. Create and attach a volume with the backing image. Verify the data content is correct.
 10. Do cleanup.
 
+### Upload via Rancher URI
+<sup>Related issue: [3129](https://github.com/longhorn/longhorn/issues/3129)</sup>
+1. Create or import cluster into Rancher
+2. Go into the cluster
+3. Click Longhorn from the left sidebar
```

Comment:
Are we assuming the Longhorn is already installed? Could you please mention that if possible a version?

Reply:
The entire test case is under `release-specific/v1.2.0/`. I think I can add a few words next to the *related issue*, so the tester will understand it which release they can go back and validate when testing latest version.

Reply:
@khushboo-rancher I will resolve this by now. As I've put Longhorn version next to the related issue, should be good hint for us to relocate these test cases in the future.